src/DropNCrop.js

```diff
@@ -166,7 +167,10 @@ class DropNCrop extends Component {
                     key="dropzone-validation"
                     className="dropzone-validation"
                   >
-                    {value && value.error}
+                    <p>{value && value.error[0]}</p>
```

Comment:
When displaying multiple error messages like this it's more flexible to just loop through all the errors and output them.

```
{value && value.error && value.error.map(error => <p>{error}</p>}
```

Also we should have scott review this to make sure he's cool with multiple errors being displayed inside `p` tags here.

Reply:
I think I'm okay with this? Would love to see what it looks like all validated out. @dmmcinty send me a screenshot ??